civicrm_resources/civicrm_resources.module

```diff
@@ -59,6 +61,46 @@ function civicrm_resources_add_resources($extention_path, $req_files = []) {
   }
 }
 
+/**
+ * Filters files into css or js
+ *
+ * @param array $files
+ * @return array
```

Comment:
Please add one empty line between `@param` and `@return` to conform to the drupal/civi code standards